stdlib/public/TensorFlow/Parameterized.swift

```diff
@@ -65,3 +65,59 @@ public extension Parameterized where Parameters : ParameterGroup {
     allParameters.update(withGradients: gradients, updater)
   }
 }
+
+//===----------------------------------------------------------------------===//
+// Extend common types to conform to ParameterGroup and Parameterized
+//===----------------------------------------------------------------------===//
+
+/// Tensors conform to the protocols by "identity".
```

Comment:
Use 2-slash comments on extensions because doc comments have no effects on extensions.
```suggestion
// Tensors conform to the protocols by "identity".
```